src/libstd/sys_common/bytestring.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(dead_code)]
```

Comment:
Is this still required?

Reply:
Yes, on windows builds the function is never called.

Reply:
Would it work if we do `#[cfg(any(unix, target_os = "redox", target_arch = "wasm32"))] pub mod bytestring` instead? I already see some of that for `pub mod gnu` and `pub mod net`. That way we continue to catch accidental dead code in this file.

Reply:
I don't think so - see `src\tools\tidy\src\pal.rs`

It looks like the goal is to avoid having platform specific code in `sys_common` and `net` is a temporary exception. The closest existing file to this is `wtf8`, which also uses `#[allow(dead_code)]`.